### PR TITLE
Update puppet gem to 3.7.0 before do provision.

### PR DESCRIPTION
**Pull request by enverromon**
Install Puppet 3.7.0 before puppet provision to prevent parsing fails.

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -1,3 +1,6 @@
-# This is the manifest used when provisiong Puppet on the Vagrant boxes.
-# Place here your Puppet test code or manage directly with the play command.
+include apt
 
+class { 'mesos':
```
Comment (alvagante) [inline manifests/init.pp]: this file  should not be changed

File: Vagrantfile [hunk 2]
```diff
@@ -1,3 +1,9 @@
+$puppet_update_to_37 = <<SCRIPT
+
+gem install puppet --version "=3.7.0" --no-ri --no-rdoc
```
Comment (alvagante) [inline Vagrantfile]: I'm not so fond of installing puppet via gems, for the possible conflicts with packaged puppet.
Would it be better to have a shell provisioner that installs puppetlabs repos and installs a recent puppet version using the local package provider?

File: manifests/init.pp (hunk 1 above)
Comment (enverromon) [inline manifests/init.pp]: Yes, this been a mistake.
Can you reject this pull request?

Comment (alvagante) [inline manifests/init.pp]: ok

**Pull request closed without merging**